tests/unit/src/Internal/Service/PaymentContextLoggerServiceTest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Class ContextLoggerServiceTest
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Tests\Internal\Service;
+
+use WCPAY_UnitTestCase;
+use WCPay\Internal\Logger;
+use WCPay\Internal\Payment\Change;
+use WCPay\Internal\Payment\PaymentContext;
+use WCPay\Internal\Payment\Transition;
+use WCPay\Internal\Service\PaymentContextLoggerService;
+use WCPay\Internal\Payment\PaymentMethod\NewPaymentMethod;
+
+/**
+ * Level3 data service unit tests.
+ */
+class PaymentContextLoggerServiceTest extends WCPAY_UnitTestCase {
+	/**
+	 * Service under test.
+	 *
+	 * @var PaymentContextLoggerService
+	 */
+	private $sut;
+
+	/**
+	 * @var Logger|MockObject
```

Comment:
A `use` is needed for `MockObject`. Otherwise (right now) that's `WCPay\Tests\Internal\Service\MockObject`, which does not exist.

Reply:
Done